android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -613,6 +618,16 @@ private void initializePlayerCore(ReactExoplayerView self) {
         self.trackSelector.setParameters(trackSelector.buildUponParameters()
                 .setMaxVideoBitrate(maxBitRate == 0 ? Integer.MAX_VALUE : maxBitRate));
 
+        Runtime runtime = Runtime.getRuntime();
```

Comment:
I don't think this change is necessary in fact...
There are some strange memory management which should not be needed